### DISCUSSION TITLE
Using placeholders on NotFound.tsx

### DISCUSSION THREAD

**Discussion by simpel** [Sitecore/jss#1796] [Help]
### Description

Hi! 

Quick question; How can we use <Placeholders> on the NotFound.tsx page. We want to show our mainmenu on the 404 page but can't figure out how to get the data in a good way.

**Comment by yavorsk** [comment]
Hi @simpel ! I think what you need is custom, content manageable 404 page that comes from sitecore. There are some comunity blog posts that explain how that can be achieved.

**Comment by pzi** [comment]
> ### Description
> Hi!
> 
> Quick question; How can we use on the NotFound.tsx page. We want to show our mainmenu on the 404 page but can't figure out how to get the data in a good way.

@simpel 

tldr; You will have to set up an error page in your Sitecore instance at a specific path. When someone punches in a route that doesn't exist in the browser and your layout service returns nothing, you gotta kick off a fetch for that error route and then return the `notFound` prop in next.js `getStaticProps/getServerSideProps`

A helpful blog-post can be found here: https://www.getfishtank.com/blog/sitecore-jss-authorable-404-page-for-nextjs-front-end
